[2021-05-25 20:56:17,955] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T20:06:59.047306+00:00 [queued]>
[2021-05-25 20:56:18,093] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T20:06:59.047306+00:00 [queued]>
[2021-05-25 20:56:18,094] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-05-25 20:56:18,094] {taskinstance.py:1043} INFO - Starting attempt 2 of 2
[2021-05-25 20:56:18,095] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-05-25 20:56:18,307] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): Get_COVID19_Cases_For_Jordan_2020ToNow> on 2021-05-25T20:06:59.047306+00:00
[2021-05-25 20:56:18,329] {standard_task_runner.py:52} INFO - Started process 130 to run task
[2021-05-25 20:56:18,351] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'covid19_cases_Jordan', 'Get_COVID19_Cases_For_Jordan_2020ToNow', '2021-05-25T20:06:59.047306+00:00', '--job-id', '108', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/workflow.py', '--cfg-path', '/tmp/tmpqvvsc1hc', '--error-file', '/tmp/tmpgjfci7y9']
[2021-05-25 20:56:18,365] {standard_task_runner.py:77} INFO - Job 108: Subtask Get_COVID19_Cases_For_Jordan_2020ToNow
[2021-05-25 20:56:18,547] {logging_mixin.py:104} INFO - Running <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T20:06:59.047306+00:00 [running]> on host 4e0af5d83701
[2021-05-25 20:56:18,689] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=waed
AIRFLOW_CTX_DAG_ID=covid19_cases_Jordan
AIRFLOW_CTX_TASK_ID=Get_COVID19_Cases_For_Jordan_2020ToNow
AIRFLOW_CTX_EXECUTION_DATE=2021-05-25T20:06:59.047306+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-05-25T20:06:59.047306+00:00
[2021-05-25 21:32:46,903] {logging_mixin.py:104} INFO - It took 36.47 minutes to get all data
[2021-05-25 21:32:50,466] {python.py:118} INFO - Done. Returned value was: None
[2021-05-25 21:32:50,541] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=covid19_cases_Jordan, task_id=Get_COVID19_Cases_For_Jordan_2020ToNow, execution_date=20210525T200659, start_date=20210525T205617, end_date=20210525T213250
[2021-05-25 21:32:50,893] {dagrun.py:497} WARNING - Failed to get task '<TaskInstance: covid19_cases_Jordan.Scoring_Report 2021-05-25 20:06:59.047306+00:00 [scheduled]>' for dag 'covid19_cases_Jordan'. Marking it as removed.
[2021-05-25 21:32:50,955] {dagrun.py:497} WARNING - Failed to get task '<TaskInstance: covid19_cases_Jordan.Save_Report_InPostgres 2021-05-25 20:06:59.047306+00:00 [None]>' for dag 'covid19_cases_Jordan'. Marking it as removed.
[2021-05-25 21:32:50,992] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-05-25 21:32:51,104] {local_task_job.py:146} INFO - Task exited with return code 0
